### PR TITLE
Add custom syslog name support

### PR DESCRIPTION
**Pull request by croessner**
You can set a custom syslog name with command line parameter "-s". The default name is "rmilter"

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -241,14 +243,39 @@ main(int argc, char *argv[])
...
+				optarg = s_pos;
+
+				syslog_name = (char *) calloc (length, sizeof (char));
+                if (syslog_name == NULL) {
+                    msg_warn ("calloc: %s", strerror (errno));
+                    return -1;
+                }
+
+				s_pos = syslog_name;
+				while (*syslog_name++ = *optarg++);
```
Comment (vstakhov) [inline src/main.c]: Why not `rmilter_stlrcpy`?

File: src/main.c [hunk 2]
```diff
@@ -241,14 +243,39 @@ main(int argc, char *argv[])
...
+				fprintf(stderr, "Illegal syslog name: %s\n",
+						optarg);
+				exit(EX_USAGE);
+			}
+			else {
+				char *s_pos;
+				size_t length = 0;
+
+				s_pos = optarg;
+				while (*optarg++) length++;
```
Comment (vstakhov) [inline src/main.c]: Why not `strlen`?

Comment (croessner) [inline src/main.c]: I grepped for string.h and it seemed to me that you did not use this import. So I thought you might have good reasons for this.

File: src/main.c (hunk 1 above)
Comment (croessner) [inline src/main.c]: I first thought about strndup, but for the same reason as above, I used my own version. But I was also not aware of rmilter_stlrcpy

File: src/main.c [hunk 3]
```diff
@@ -60,6 +60,7 @@ usage (void)
 			"-d - debug parsing\n"
 			"-h - this help message\n"
 			"-c - path to config file\n"
+			"-s - syslog name (defaults to rmilter)\n",
```
Comment (croessner) [inline src/main.c]: Or maybe both cmd and cfg

**Pull request merged**